layouts/default.vue

```diff
@@ -15,6 +15,12 @@ export default {
         this.$router,
       );
     }
+    if (/mobile/i.test(navigator.userAgent) && useRoute().fullPath != "/") {
+      alert(
+        "We are sorry to inform you that this game is not optimized for mobile devices. Please use a desktop browser.",
+      );
+      this.$router.push("/");
+    }
```

Comment:
https://nuxt.com/docs/getting-started/routing#route-middleware
du baust middleware nach damit, also es ist so wie in den letzten prs recht schirch geschrieben

wenn dann mach's wenigstens in app.vue